sim/scenarios/droplist/README.md

```diff
@@ -0,0 +1,38 @@
+# Droplist
+
+This scenario uses a bottleneck link similar to the [../simple-p2p](simple-p2p)
+scenario and optionally allows configuring the link to drop specific packets in
+either direction.
+
+This scenario can be used to test the performance of QUIC implementations under
+specific loss patterns. For instance, tests can:
+* drop specific handshake packets,
+* drop tail packets,
```

Comment:
How would you identify a tail packet?

Reply:
You would have to run the simulation once with no drops, find the packet number of the tail packet (manually), and then give that as a packet number for a rerun.